src/html/extractors/factories/elementAttribute.ts

```diff
@@ -5,8 +5,13 @@ import { Validate } from '../../../utils/validate';
 import { IHtmlExtractorOptions, validateOptions } from '../common';
 import { IContentOptions, getContentOptions, validateContentOptions } from '../../../utils/content';
 
+/**
+ * elementAttributeExtractor extractor messages from element attribute
+ *
+ *     <translate context='ctx' text='msgid' plural='plural msg'></translate>
+ */
 export function elementAttributeExtractor(selector: string, textAttribute: string, options: IHtmlExtractorOptions = {}): IHtmlExtractorFunction {
-    Validate.required.nonEmptyString({selector, textAttribute});
+    Validate.required.nonEmptyString({ selector, textAttribute });
```

Comment:
Please refrain from changing parts of the code unrelated to your PR (same goes for changes in elementContent and embeddedJs)